_includes/footer.html

```diff
@@ -1,4 +1,9 @@
-<p class="site-footer">
+<div class="site-footer">
+  <ul>
+    <li><a href="/about">About</a></li>
```

Comment:
To look perfect, these all have to be on the same line, which is kinda butts, but… 🤷‍♂️